user_manual/groupware/mail.rst

```diff
@@ -10,12 +10,14 @@ Using the Mail app
 Managing your mail account
 ---------------------------
 
-Switch layout
-~~~~~~~~~~~~~
+Switch mailing list layout
```

Comment:
```suggestion
Switch layout
```

To not cause confusion with *mailing lists* https://en.wikipedia.org/wiki/Mailing_list